src/directives/public/for.js

```diff
@@ -513,6 +514,23 @@ module.exports = {
         frag.destroy()
       }
     }
+  },
+
+  _bubbleChanges: function (scope, value) {
```

Comment:
Not sure about the method name here. There's no "bubbling" here - maybe something like `syncChanges`?